### DISCUSSION TITLE
[FEATURE REQUEST] security metadata in repo

### DISCUSSION THREAD

**Discussion by ssoto2** [saltstack/salt#67683] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
Recently there where SaltStack CVE's that came out. These were talked about widely both by press but by saltstack itself https://community.saltstack.com/blog/critical-vulnerabilities-update-cve-2020-11651-and-cve-2020-11652/ 

While the Vuln was listed on the website automated tools like yum did not see that the update was a security update

**Describe the solution you'd like**
add security metadata to the repo data so tools like spacewalk can pick up that there is a security update or for systems that yum security

**Describe alternatives you've considered**
The need to purchase alternative tools just to know when security errata have been found, or search websites for this data

**Additional context**
https://linux.die.net/man/8/yum-security

**Please Note**
If this feature request would be considered a substantial change or addition, this should go through a SEP process here https://github.com/saltstack/salt-enhancement-proposals, instead of a feature request.

**Comment by garethgreenaway** [comment]
@saltstack/team-packaging FYI.

**Comment by bryceml** [comment]
https://access.redhat.com/solutions/23016#security looks helpful on how to add it to a repo.

I'm having a hard time finding any tools for creating the updateinfo.xml file in an automated way.

@dmurphy18 any ideas on tools for this?

**Comment by bryceml** [comment]
Also, the way we do packages now, where we create a repo and then never touch it after it's released doesn't really work for this.  It could work for people pointing to a major release or latest.

I think it makes the most sense to switch to an all in 1 repo per distro version with all versions in the repo, and then package managers could always get up to date metadata.
